app/controllers/greetings_controller.rb

```diff
@@ -0,0 +1,6 @@
+# Landing page
+class GreetingsController < ApplicationController
+  def hello
```

Comment:
In order to be more RESTful in the resource and routes definition, this action could be named `index` as a means to represent the greetings index page.